bin/dune

```diff
@@ -1,7 +1,7 @@
 (executables
  (names main)
  (libraries fialyzer bitstring obeam)
- (preprocess (pps ppx_deriving.std)))
+ (preprocess (pps ppx_deriving.std ppx_here)))
```

Comment:
Could you add `ppx_here` to `fialyzer.opam` as a dependency?

Reply:
`ppx_here` is also required from obeam: https://opam.ocaml.org/packages/obeam/obeam.0.1.1/
So it is not necessary to write `fialyzer.opam`.  Is it preferred to specify it?

Reply:
You are right! I fixed.
Thank you.